Core/DISET/private/FileHelper.py

```diff
@@ -27,375 +28,375 @@ def __init__( self, oTransport = None, checkSum = True ):
     return result
 
-  def __createTar( self, fileList, wPipe, compress, autoClose = True ):
-    if 'write' in dir( wPipe ):
+  def __createTar(self, fileList, wPipe, compress, autoClose=True):
+    if 'write' in dir(wPipe):
       filePipe = wPipe
     else:
-      filePipe = os.fdopen( wPipe, "w" )
+      filePipe = os.fdopen(wPipe, "w")
     tarMode = "w|"
     if compress:
       tarMode = "w|bz2"
 
-    with tarfile.open( name = "Pipe", mode = tarMode, fileobj = filePipe ) as tar:
+    with tarfile.open(name="Pipe", mode=tarMode, fileobj=filePipe) as tar:
       for entry in fileList:
-        tar.add( os.path.realpath( entry ), os.path.basename( entry ), recursive = True )
+        tar.add(os.path.realpath(entry), os.path.basename(entry), recursive=True)
     if autoClose:
       try:
         filePipe.close()
-      except:
+      except BaseException:
         pass
 
-  def bulkToNetwork( self, fileList, compress = True, onthefly = True ):
+  def bulkToNetwork(self, fileList, compress=True, onthefly=True):
     if not onthefly:
       try:
         filePipe, filePath = tempfile.mkstemp()
       except Exception as e:
-        return S_ERROR( "Can't create temporary file to pregenerate the bulk: %s" % str( e ) )
-      self.__createTar( fileList, filePipe, compress )
+        return S_ERROR("Can't create temporary file to pregenerate the bulk: %s" % str(e))
+      self.__createTar(fileList, filePipe, compress)
       try:
         fo = open(filePath, 'rb')
       except Exception as e:
-        return S_ERROR( "Can't read pregenerated bulk: %s" % str( e ) )
-      result = self.DataSourceToNetwork( fo )
+        return S_ERROR("Can't read pregenerated bulk: %s" % str(e))
+      result = self.DataSourceToNetwork(fo)
       try:
         fo.close()
-        os.unlink( filePath )
-      except:
+        os.unlink(filePath)
+      except BaseException:
         pass
       return result
     else:
       rPipe, wPipe = os.pipe()
-      thrd = threading.Thread( target = self.__createTar, args = ( fileList, wPipe, compress ) )
+      thrd = threading.Thread(target=self.__createTar, args=(fileList, wPipe, compress))
       thrd.start()
-      response = self.FDToNetwork( rPipe )
+      response = self.FDToNetwork(rPipe)
       try:
-        os.close( rPipe )
-      except:
+        os.close(rPipe)
+      except BaseException:
         pass
       return response
 
-  def __extractTar( self, destDir, rPipe, compress ):
-    filePipe = os.fdopen( rPipe, "r" )
+  def __extractTar(self, destDir, rPipe, compress):
+    filePipe = os.fdopen(rPipe, "r")
     tarMode = "r|*"
     if compress:
       tarMode = "r|bz2"
-    with tarfile.open( mode = tarMode, fileobj = filePipe ) as tar:
+    with tarfile.open(mode=tarMode, fileobj=filePipe) as tar:
       for tarInfo in tar:
-        tar.extract( tarInfo, destDir )
+        tar.extract(tarInfo, destDir)
     try:
       filePipe.close()
-    except:
+    except BaseException:
```

Comment:
Why `BaseException`? (here and in the other spaces)

Reply:
Mistake. I will correct them